### DISCUSSION TITLE
my account is flagged. But why I have not done any violation. It is me.

### DISCUSSION THREAD

**Discussion by arunguptaag** [community/community#52385] [New to GitHub]
Hi I would like to remove flagged from my account

**Comment by queenofcorgis** [comment]
Hi @arunguptaag ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) As the community is unable to assist in security matters like this I'll be closing and locking the post.
